recipe/meta.yaml

```diff
@@ -1,29 +1,32 @@
 {% set name = "hydra-core" %}
-{% set version = "1.0.6" %}
+{% set version = "1.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/facebookresearch/hydra/archive/v{{ version }}.tar.gz
-  sha256: fb1ad4f26332d0d88a180731204a42aed596e3edc0627b6c45cabb40fb1cf9b7
+  sha256: 5a50c0eeadcad4324a6bd895b6dcbe17068558143c8080e1cc6cdbe637c679de
 
 build:
   noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: 
+    - {{ PYTHON }} setup.py antlr
+    - {{ PYTHON }} -m pip install . -vv
```

Comment:
As far as I know, pip install should run antlr. Did did you see otherwise?

Reply:
This was failing without line 16. I'll remove it to double check.

Reply:
@omry https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=349029&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=797

Reply:
Odd. what command is it running?
I never need to run the antlr command directly locally, `pip install .` do the trick.

Reply:
I'm not sure outside of what is shown in the logs. Do you think it will cause an issue?

Reply:
No, it should be okay to trigger it manually.